exceptions_hog/handler.py

```diff
@@ -143,17 +161,39 @@ def _get_detail(exc, exception_key: str = "") -> str:
             )  # We do str() to get the actual error string on ErrorDetail instances
         elif isinstance(exc.detail, dict):
             value = exc.detail
-            for key in exception_key.split(api_settings.NESTED_KEY_SEPARATOR):
+
+            # Handle nested attributes
+            for key in exception_key:
```

Comment:
What happens here `exception_key` is `str`? The type `Union[str, List[str]]` makes me unsure

Reply:
Great point, added an extra check!